components/Card.js

```diff
@@ -5,7 +5,12 @@ import Image from 'next/image'
  */
 export default function Card(props) {
   return (
-    <div className="h-auto w-48 mx-auto md:w-44">
+    <div
+      className={`h-auto w-48 mx-auto md:w-44 ${
+        props.selected ? `border border-red-500` : ``
+      }`}
```

Comment:
The border isn't turning red :(

Reply:
if you add `border border-slate-300` to the else rather than blank, the cards won't shrink when applying the border (once it's working 😉 )